avm/res/network/nat-gateway/main.bicep

```diff
@@ -99,7 +110,7 @@ module publicIPAddresses 'br/public:avm/res/network/public-ip-address:0.2.1' = [
       skuName: 'Standard' // Must be standard
       skuTier: publicIPAddressObject.?skuTier
       tags: publicIPAddressObject.?tags ?? tags
-      zones: publicIPAddressObject.?zones ?? (zone != 0 ? [string(zone)] : null)
+      zones: publicIPAddressObject.?zones
```

Comment:
How come? 0 not an option anymore?

Reply:
Hi, sorry, removed by mistake. I restored it, however without conversion to a string, corresponding to the referenced version 0.5.1 of the PIP module.